addons/mail/static/tests/legacy/message/message_tests.js

```diff
@@ -1689,3 +1689,36 @@ QUnit.test("Click on view reactions shows the reactions on the message", async (
     await click(".o-mail-Message-moreMenu [title='View Reactions']");
     await contains(".o-mail-MessageReactionMenu", { text: "😅1" });
 });
+
+QUnit.test("discuss - bigger font size when there is only emoji", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "channel",
+        name: "channel1",
+    });
+    const { openDiscuss } = await start();
+    await openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", "🥳");
+    await click(".o-mail-Composer-send:enabled");
+    await contains(".o-mail-Message-body.fs-1", { text: "🥳" });
+    await insertText(".o-mail-Composer-input", "not only emoji!! 😅");
+    await click(".o-mail-Composer-send:enabled");
+    await contains(".o-mail-Message-body:not(.fs-1)", { text: "not only emoji!! 😅" });
+});
+
+QUnit.test("chatter - font size unchanged when there is only emoji", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "channel",
+        name: "channel1",
+    });
+    const { openDiscuss, openFormView } = await start();
+    await openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", "🥳");
+    await click(".o-mail-Composer-send:enabled");
+    await openFormView("res.partner", pyEnv.currentPartnerId);
```

Comment:
The steps related to `openDiscuss()` in the "chatter" test look irrelevant

Reply:
it's better to:
- Open discuss and check that the emoji is bigger
- Open the chatter then check its not the case
Otherwise we would have a negative check only which is not reliable